spellbook/models/uniswap/ethereum/uniswap_v3_ethereum_trades.sql

```diff
@@ -3,7 +3,7 @@
 }}
         
 SELECT
-    tx_hash || '-' || evt_index::string as unique_trade_id,
+    'v3' || '-' || tx_hash || '-' || evt_index::string as unique_trade_id,
```

Comment:
Feels weird to me that it was the overlap between v2 and v3 that caused duplicates on unique_trade_id